### DISCUSSION TITLE
Thank you, again.

### DISCUSSION THREAD

**Discussion by Nothoro** [otya128/winevdm#1011] [General]
I wrote you a couple years ago regarding this project and saying that you've done what Microsoft has either refused or neglected to do. I just want to thank you again for all of your hard work in this project. It works great and is an awesome contribution to the community.
Thanks again.
Best Regards, Brian.

**Comment by cnfcomps** [comment]
I would just like to also add my thanks as I have been able to install an old piece of software that used a 16-Bit installer on Windows 10 64-Bit!
Matt.

**Comment by TurboBlaise** [comment]
A Million thanks for your great work!
Cracyc, to solve the problem with the laguage.dll within just one day shows your enthusiastic commitment and enables the use of Borland's ObjectVision with full database access on 64-bit-systems. 
You fulfill what M$ has forgotten. As I said in the other thread: **Otya and Cracyc are the heroes of the lost 16-bit-apps!**

**Comment by Eksol** [comment]
Помогло!!! Запустился старый установщик на 16 бит. Огромное спасибо!!!
